commands/build.go

```diff
@@ -56,6 +56,25 @@ func (b *buildcmd) build(c *cli.Context) error {
 
 	dir := common.GetDir(c)
 
+	ffV, err := common.ReadInFuncFile()
+	version := common.GetFuncYamlVersion(ffV)
+	if version == common.LatestYamlVersion {
```

Comment:
This would be better as a 'switch' statement.

switch version:
v2018: blah
v1: blah